ui/src/lib/helpers.tsx

```diff
@@ -338,4 +338,10 @@ export const getDOIsFromText = (text: string) => text.match(/(10\.[0-9a-zA-Z]+\/
 
 export const validateDOI = (value: string) => /(10\.[0-9a-zA-Z]+\/(?:(?!["&\'])\S)+)\b/.test(value);
 
+export const validateEmail = (email: string): Boolean => {
```

Comment:
might be useful to have some actual tests to test some valid and invalid emails

Reply:
could be a unit test for this method (though we don't have anything like that at the moment but we probably should) or indirectly via a test for the CoAuthor component